server/vmap/trillian_map_server.go

```diff
@@ -37,12 +34,12 @@ func (t *TrillianMapServer) getHasherForMap(mapID int64) (merkle.MapHasher, erro
 // GetLeaves implements the GetLeaves RPC method.
 func (t *TrillianMapServer) GetLeaves(ctx context.Context, req *trillian.GetMapLeavesRequest) (resp *trillian.GetMapLeavesResponse, err error) {
 	ctx = util.NewMapContext(ctx, req.MapId)
-	s, err := t.registry.GetMapStorage(req.MapId)
+	s, err := t.registry.GetMapStorage()
```

Comment:
Please move GetMapStorage to the constructor now that MapID is not longer required for instantiation.